package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -337,64 +356,78 @@ public class GeometrySubsets {
           // TODO: this should be moved to C++
           int[] offsetMapping = new int[usdMesh.faceVertexCounts.Length];
           int curOffset = 0;
-          for (int i = 0; i < usdMesh.faceVertexCounts.Length; i++) {
+          for (int i = 0; i < usdMesh.faceVertexCounts.Length; i++)
+          {
```

Comment:
Curly braces on new line is actually the Unity style, but I would make that update in a separate change (across the entire code base), since it makes it harder to see the diff in this PR

Reply:
I know, I'm sorry, I did it for readability when I worked on it, but I should have removed it before doing the PR... I was lazy... I'll do it in my next commit :)